CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to GerryChainJulia
+
+Thank you for your interest in contributing to GerryChainJulia! Please use the following guidelines on how to contribute to this project:
+
+## How can I contribute?
+
+### Reporting bugs
+
+If you find a bug while using any tools in this repository, consider creating a [GitHub issue](https://github.com/mggg/GerryChainJulia/issues) documenting the problem. Be sure to be detailed. Explain what you did, what you expected to happen, and what actually happened.
+
+### Updating documentation
+
+If you find any documentation that is lacking or in error, submit a pull request that updates it. 
+
+### Suggesting enhancements
```

Comment:
I like the way you separated out the process for reporting bugs & suggesting enhancements! 👍